app/scripts/services/Room.js

```diff
@@ -3,8 +3,14 @@
 		var ref = firebase.database().ref().child("rooms");
 		var rooms = $firebaseArray(ref);
 
+		var addRoom = function(roomName) {
+			rooms.$add(roomName);
```

Comment:
This is not the correct syntax for `$add`, it expects an object. See https://github.com/firebase/angularfire/blob/master/docs/reference.md#addnewdata